### ISSUE TITLE
Skip ignored when empty reason provided

### ISSUE THREAD

**Issue by ltickett** [xunit/xunit#2286]
I did something like this (because I am lazy):
`
        [Fact(Skip = "")]
        public void Send()
        {

        }
`

And the test was still executed. Should we either not allow the reason to be left blank/empty? Or skip it regardless of whether it is blank/empty?

**Comment by bradwilson** [comment]
Good catch, this shouldn't run. I'm not sure whether it should reject the test, or provide an empty message like `no skip message provided`.

**Comment by ltickett** [comment]
@bradwilson let me know if this is something I can attempt to contribute. I've never contributed to the project before but would love to have a stab.

Thanks

**Comment by bradwilson** [comment]
This is working as expected in v3, albeit with an empty message. Open a new issue if you think it should get a default message instead.

![Image](https://github.com/user-attachments/assets/b6cae6a0-a4a9-4afc-b742-9885424aeaef)
